homeworks/artem.sokur_Grimdvl/PokemonProject/pokemons.html

```diff
@@ -0,0 +1,365 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="https://fonts.google.com/share?selection.family=Roboto:wght@100;300;700">
+        <link rel="stylesheet" href="style/style.css">
+        <link rel="stylesheet" href="style/pokemons.css">
+        <title>PokemonProject</title>
+    </head>
+    <body>
+        <header>
+            <div class="container" style="background-color: #333;">
+                <section class="head">
+                    <a class="head__logo" href="https://www.mastersacademy.education/" target="_blank">
+                        <img class="head__logo-img" src="img/icons/logo-ma.svg" alt="masters-academy">
+                        <span class="head__logo-title">MASTERS ACADEMY</span>
+                    </a>
+    
+                    <nav class="head__navigation">
+                        <a class="head__navigation-item active" href="#">Pokemons</a>
+                        <a class="head__navigation-item" href="index.html">About</a>
+                        <a class="head__navigation-item" href="login.html">Login</a>
+                    </nav>
+                </section>
+            </div>
+
+            <div class="container" style="background-color: #221F1F;">
+                <section class="filters">
+                    <form class="filters__drop">
```

Comment:
pls use one form tag for all items in this panel!

Reply:
I mean this!
<img width="334" alt="Screenshot 2023-10-25 at 21 21 33" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/115080771/af8ed934-33c2-4ef0-8c79-1d67d99f82e5">